tests/components/observers/test_household_survey_observers.py

```diff
@@ -0,0 +1,63 @@
+import tempfile
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from vivarium_census_prl_synth_pop.components.observers import HouseholdSurveyObserver
+
+# TODO: Broader test coverage
+
+
+@pytest.fixture
+def observer(mocker):
+    """Generate post-setup observer with mocked methods to patch as necessary"""
+    observer = HouseholdSurveyObserver("ACS")
+    builder = mocker.MagicMock()
+
+    # create a temp directory so setup can generate an output directory
+    tmpdir = tempfile.TemporaryDirectory()
+    builder.configuration.output_data.results_directory = tmpdir.name
+    observer.setup(builder)
+    tmpdir.cleanup()
+    return observer
+
+
+@pytest.fixture
+def mocked_pop_view():
+    """Generate a state table view"""
+    cols = HouseholdSurveyObserver.INPUT_COLS
+    df = pd.DataFrame(np.random.randint(0, 100, size=(2, len(cols))), columns=cols)
+    df["alive"] = "alive"
+    df[["first_name", "middle_name", "last_name"]] = (
+        ["Alex", "J", "Honnold"],
```

Comment:
Haha love the names

Reply:
Interestingly, Honnold actually does not have a middle name. But in some interview he claimed his middle name is "J".